be/src/runtime/runtime_state.cpp

```diff
@@ -531,5 +531,14 @@ Status RuntimeState::get_codegen(LlvmCodeGen** codegen) {
 Status RuntimeState::StartSpilling(MemTracker* mem_tracker) {
     return Status::InternalError("Mem limit exceeded.");
 }
+
+int64_t RuntimeState::get_load_mem_limit() {
+    if (_query_options.__isset.load_mem_limit && _query_options.load_mem_limit > 0) {
```

Comment:
I'm afriad that I don't find that you set this value

Reply:
In constructor of Coordinator in FE.
`this.queryOptions = context.getSessionVariable().toThrift();`

Reply:
Oh, I see. However I think it's not a good idea to get all queryOptions from session variables. For example, if we intent to support setting this value in load statement other than through setting variables, then it is not available to get right limit from session variables.
And I think it's better to support this option in stream load.

Reply:
For broker and stream load, user can set mem limit by `exec_mem_limit` property wrote in load stmt or http request. And I will use `exec_mem_limit` to overwrite `load_mem_limit` in `queryOptions` in query plan.

That is to say, for broker and stream load, user no need to care about this `load_mem_limit`,
and this new variables will make no effect on these kind of load.